Update tutorial-custom-images.md

pngfei opened the pull request:
It would be better to set image location to westeurope, as the source VM's location is westeurope in sample script of VM creation

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

pngfei committed:
Update tutorial-custom-images.md

It would be better to set image location to westeurope, as the source VM's location is westeurope in sample script of VM creation
